buildSrc/src/main/groovy/org/apereo/gradle/plugin/CasPlugin.groovy

```diff
@@ -13,10 +13,17 @@ import java.security.MessageDigest
  * CAS Gradle plugin implementing user-friendly DSL for building and deploying Apereo CAS server.
  *
  * @author Jonathan Johnson
+ * @author Dmitriy Kopylenko
  * @since 1.0.0
  */
 class CasPlugin implements Plugin<Project> {
 
+    static final CAS_ORIG_SUFFIX = '.casorig'
+
+    static final RESOURCES_DIR = 'src/main/resources'
+
+    static final SRC_DIR = 'src'
```

Comment:
this might be a little aggressive.. we might only want to use something like `src/main/resources` for this one as well, and leave other empty directories alone. Just a thought